frontend/test/unit/setup-after-env.js

```diff
@@ -1,5 +1,8 @@
 import Vue from "vue"
 import "@testing-library/jest-dom"
+import failOnConsole from "jest-fail-on-console"
+
+failOnConsole()
```

Comment:
Will this also catch output from our custom console utils? They indirectly refer to the inbuilt `console` functions but I'm asking just to be sure.

Reply:
Yup! The library is implemented by modifying the global console object so unless we were doing something like writing directly to `stdout`, we wouldn't be able to easily _accidentally_ bypass this utility.

https://github.com/ValentinH/jest-fail-on-console/blob/main/index.js#L102-L115